docs/tutorials/storage.md

```diff
@@ -60,4 +60,4 @@ config.video[0].storage.identifier = manager.select( acquire.DeviceKind.Storage,
 config.video[0].storage.settings.filename = "out.tif" 
 ```
 
-Before proceeding, complete the `Camera` setup and call `runtime.set_configuration(config)` to save those new configuration settings.
\ No newline at end of file
+Before proceeding, complete the `Camera` setup and call `set_configuration` to save those new configuration settings.

```

Comment:
Why this change?

Reply:
I wanted to keep the tutorials short. I can add a reference to another tutorial that describes how to setup the camera.

Reply:
To match formatting in other tutorials